client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -59,6 +53,33 @@ const enableImageEditing = ( item, site ) => {
 	return true;
 };
 
+/**
+ * This function return true if the video editor can be
+ * enabled/shown
+ *
+ * @param  {object} item - media item
+ * @param  {object} site - current site
+ * @return {boolean} `true` if the video editor can be enabled.
+ */
+const enableVideoEditing = ( item, site ) => {
+	// do not show if the feature flag isn't set
+	if ( ! config.isEnabled( 'post-editor/video-editor' ) ) {
+		return false;
+	}
+
+	// do not allow for Jetpack site with VideoPress disabled
+	if ( get( site, 'jetpack', false ) ) {
```

Comment:
Actually, there are selectors for checking if a Jetpack site and if VideoPress is enabled. Switched to using them in https://github.com/Automattic/wp-calypso/pull/11653/commits/3817ac4d102e0baeadd7021d2df43a3235fbd301.